plugin/storage/badger/samplingstore/storage_test.go

```diff
@@ -0,0 +1,104 @@
+// Copyright (c) 2023 The Jaeger Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package samplingstore
+
+import (
+	"testing"
+	"time"
+
+	"github.com/dgraph-io/badger/v3"
+	"github.com/stretchr/testify/assert"
+
+	samplemodel "github.com/jaegertracing/jaeger/cmd/collector/app/sampling/model"
+)
+
+type samplingStoreTest struct {
```

Comment:
why is the wrapper necessary? You could pass `*SamplingStore` to test lambdas directly, less complexity